subsys/partition_manager/pm.yml.settings

```diff
@@ -8,6 +8,8 @@ settings_storage:
 #else
 #ifdef CONFIG_BUILD_WITH_TFM
     align: {start: CONFIG_NRF_SPU_FLASH_REGION_SIZE}
+#else
+    align: {start: CONFIG_FPROTECT_BLOCK_SIZE}
```

Comment:
```suggestion
    align: {start: CONFIG_NRF_SETTINGS_STORAGE_ALIGN}
```

And then add some kconfig

```
config NRF_SETTINGS_STORAGE_ALIGN
      int "Storage partition alignment"
      default SPU_FLASH_REGION_SIZE if BUILD_WITH_TFM
      default FPROTECT_BLOCK_SIZE
      help
        Alignment of storage partition.
```

That way if users for some reason don't want to align on fprotect boundaries (maybe they only want page alignment) they can get that as well.

Reply:
Thanks for the feedback. Your solution seems more elegant and flexible for the user. There is still an open question of where we should have such a Kconfig. I would suggest having it in the Kconfig of the PM. There is already a Settings configuration. To follow the naming in the PM Kconfig, I would use the following name:

```config PM_PARTITION_ALIGN_SETTINGS_STORAGE```

Do you agree with this approach?

Reply:
It seems that we have other similar options in this file (`subsys/partition_manager/Kconfig`) so placing it there seems sane to me. Changing the name to match is also welcome.

Reply:
Thanks for the input. I addressed your review comments